server/notification.js

```diff
@@ -219,13 +212,10 @@ class Notification {
 /**
  * Apply the notification to every monitor
  * @param {number} notificationID ID of notification to apply
- * @param {number} userID ID of user who created notification
  * @returns {Promise<void>}
  */
-async function applyNotificationEveryMonitor(notificationID, userID) {
-    let monitors = await R.getAll("SELECT id FROM monitor WHERE user_id = ?", [
-        userID
-    ]);
+async function applyNotificationEveryMonitor(notificationID) {
+    let monitors = await R.getAll("SELECT id FROM monitor", []);
```

Comment:
empty [] is not needed in this case. See https://redbean-node.whatsticker.online/Query

```suggestion
    let monitors = await R.getAll("SELECT id FROM monitor");
```